net/crowdsec-firewall-bouncer/Makefile

```diff
@@ -6,12 +6,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=crowdsec-firewall-bouncer
-PKG_VERSION:=0.0.31
-PKG_RELEASE:=2
+PKG_REALVERSION:=0.0.32-rc1
+PKG_VERSION:=0.0.32_rc1
```

Comment:
If `rc1` stands for a release candidate, then why not wait for the release?

Reply:
Actually I'm running the rc1. I assume others running current OpenWrt master would also like to participate. 
For 24.10 branch it will only be the release version.

Reply:
Updated to release version 0.0.32